lib/content_importer.rb

```diff
@@ -99,4 +101,11 @@ def overwrite_locale_links(input_locale_path, locale_hash, output_locale_path =
       file.write(existing_locale_file.to_yaml)
     end
   end
+
+  def blank_row?(row)
+    row.each do |_heading, value|
+      return false unless [nil, "#N/A"].include? value
+    end
+    true
```

Comment:
I think a more idiomatic way to say this would be
```ruby
def blank_row?(row) row.values.any? { |v| [nil, "#N/A"].include?(v) }; end
```

Reply:
Originally I found there wasn't a `values` method but after searching again I realised `fields` gives us what we need 👍

Reply:
Ah cool, thanks 👍